libs/sdk-ui-filters/src/DateFilter/DateRangePicker/DateRangePicker.tsx

```diff
@@ -150,6 +153,26 @@ class DateRangePickerComponent extends React.Component<DateRangePickerProps, IDa
             </div>
         );
 
+        const OverlayDatePicker = (
+            <Overlay
+                alignTo={`.gd-date-range-picker-${isTimeEnabled ? this.state.selectedInput : "from"}`}
+                alignPoints={[
```

Comment:
Please extract this object as constant outside of the main component.

Reply:
Done